src/popup/utils/helper.js

```diff
@@ -174,6 +173,9 @@ const initializeSDK = (ctx, { network, current, account, wallet, activeAccount =
                 resolve(sdk)
             }
         })
+        // .catch(err => {
```

Comment:
Why is this commented out?

Reply:
This was the only way to open the extension when the following error occured
![viber_image_2019-07-31_12-02-59](https://user-images.githubusercontent.com/43376240/62864915-afa06c80-bd15-11e9-828f-8d789a56651d.jpg)

I will remove it now

Reply:
It is now obsolete as the error is fixed. Thanks!